assets/js/atomic/utils/render-parent-block.tsx

```diff
@@ -138,16 +138,13 @@ const renderInnerBlocks = ( {
 			const parsedElement = parse( element.outerHTML );
 
 			if ( isValidElement( parsedElement ) ) {
-				const elementChildren =
-					element.children && element.children.length
```

Comment:
We skipped this condition because we thought it was the issue with rendering, should we put it back?

Reply:
restoring them didn't break checkout automatic migration on frontend.

Reply:
I think leave it. It works either way and this will ensure forced blocks always append.